src/AppInstallerCommonCore/Runtime.cpp

```diff
@@ -309,6 +312,36 @@ namespace AppInstaller::Runtime
                 result = GetKnownFolderPath(FOLDERID_Profile);
                 create = false;
                 break;
+            case PathName::PortableAppUserRoot:
+                result = Settings::User().Get<Setting::PortableAppUserRoot>();
+                if (result.empty())
+                {
+                    result /= GetKnownFolderPath(FOLDERID_LocalAppData);

```

Comment:
```suggestion
                    result = GetKnownFolderPath(FOLDERID_LocalAppData);
```
Even though result was just verified to be empty, it feels unsafe to add a path to it.

Reply:
Changed to suggested